knative-kubecon/build/020-serving.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: helloworld-openshift-python
+  namespace: myproject
+spec:
+  runLatest:
+    configuration:
+      build:
+        source:
+          git:
+            url: https://gitlab.com/vdemeester/os-sample-python.git
+            revision: master
+        template:
+          name: openshift-builds # change that
+          arguments:
+          - name: IMAGE_STREAM
+            value: python:3.6
+          - name: IMAGE
+            value: "helloworld:latest"
+          - name: NAME
+            value: foo-build
+      revisionTemplate:
+        metadata:
+          annotations:
+            alpha.image.policy.openshift.io/resolve-names: "*"
+        spec:
+          container:
```

Comment:
can we add `containerConcurrency: 1` above, so we "force" autoscaling to work a bit harder (makes for an easier demo)

Reply:
updated :wink: